zuul.d/secrets.yaml

```diff
@@ -95,4 +95,18 @@
         qQmYw+ZF9T42q7Kn587UYCpoU0x0eOUgbvu16ZrgJR67vDQgWG/gP10wc1UePgApMC/kr
         Bh1Bqt/7TpTQGKx3765xbC3BNEUOy6ayqSmG6N8dsuGRoSKeFLS8wctHq9AaH03IXFI0W
         b6S6T6vBW1XNUsqCk3Y3rvmk3/pyimsEz0TUU8MF9AMFCl7YAuHmvggEUNVvt9uNXr9qB
-        ua2JBCYeuiskHM97rTnEW3k6dDrWNytR/NCm4UH9JzQhW99NOUFxgwkoXrCTok=
\ No newline at end of file
+        ua2JBCYeuiskHM97rTnEW3k6dDrWNytR/NCm4UH9JzQhW99NOUFxgwkoXrCTok=
+- secret:
+    name: netapp_vsadmin_password
+    data:
+      evalue: !encrypted/pkcs1-oaep
```

Comment:
Should this be value or evalue? I notice that the other keys are value

Reply:
Good spot, its a typo brought over from a typo in the zuul-client command.